cmd/mirror-main.go

```diff
@@ -71,13 +71,13 @@ var (
 			Name:  "exclude",
 			Usage: "exclude object(s) that match specified object name pattern",
 		},
-		cli.IntFlag{
+		cli.StringFlag{
 			Name:  "older-than",
-			Usage: "filter object(s) older than N days",
+			Usage: "filter object(s) older than L days M hours and N minutes",
```

Comment:
Should `Usage: "filter object(s) older than L days M hours and N minutes",` be `Usage: "filter object(s) older than L days, M hours and N minutes",`